lib/Model/DBConnect.php

```diff
@@ -11,10 +11,10 @@ class DBConnect
 
     public function connect()
```

Comment:
WARNING !! do not load this file online with sensitive data in it ! We will talk about it.